packages/cli/src/commands/deploy/latest.ts

```diff
@@ -176,7 +189,7 @@ const printDeploymentStatus = async (
 };
 
 // Converts `env` Arrays, Strings and Objects into env Objects.
-const parseEnv = env => {
+const parseEnv = (env: any | object) => {
```

Comment:
```suggestion
const parseEnv = (env?: string[] | { [key: string]: string}) => {
```

Reply:
@styfle  This solution is not working. I tried it in my previous PR

Reply:
Added proper types here.